app/src/main/java/dev/arkbuilders/arkmemo/ui/fragments/EditTextNotesFragment.kt

```diff
@@ -74,6 +63,9 @@ class EditTextNotesFragment: Fragment(R.layout.fragment_edit_notes) {
 
             override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
                 title = s?.toString() ?: ""
+                if (title.isEmpty()) {
+                    binding.noteTitle.hint = getString(R.string.hint_new_text_note)
```

Comment:
I think its better to have timestamp on the default tilte to make it easy for user to identify a specfic note among notes saved with default titles. What do you think @kirillt

Reply:
@ShubertMunthali We're still showing note's timestamp which is taken from `modified` property of the note's resource field.

<img width="411" alt="Screenshot 2024-01-09 at 23 03 06" src="https://github.com/ARK-Builders/ARK-Memo/assets/4281969/8c794298-b181-4352-9b9e-65ea0321dbb9">

Reply:
@ShubertMunthali I think we don't need the timestamp in the title, but we should have it somewhere around.

@tuancoltech looks good, but this timestamp can be updated externally. We need https://github.com/ARK-Builders/arklib-android/issues/135 for proper UX (so the timestamp would stick to the resource forever).

Reply:
@kirillt Sure, let's get back to this after we have `created` date thing ready.